package-lock.json

```diff
@@ -1541,23 +1541,6 @@
         "url": "https://opencollective.com/typescript-eslint"
       }
     },
-    "node_modules/@typescript-eslint/scope-manager": {
```

Comment:
Do you know why there are a bunch of deletions here?

Reply:
I think this had something to do with dependabot. After the last batch of dependabot updates, I pulled down the main branch and ran `npm i` -- it resulted in all of these changes to the package-lock